juniper/src/integrations/jiff.rs

```diff
@@ -2,26 +2,26 @@
 //!
 //! # Supported types
 //!
-//! | Rust type           | Format                | GraphQL scalar        |
-//! |---------------------|-----------------------|-----------------------|
-//! | [`civil::Date`]     | `yyyy-MM-dd`          | [`LocalDate`][s1]     |
-//! | [`civil::Time`]     | `HH:mm[:ss[.SSS]]`    | [`LocalTime`][s2]     |
-//! | [`civil::DateTime`] | `yyyy-MM-ddTHH:mm:ss` | [`LocalDateTime`][s3] |
-//! | [`Timestamp`]       | [RFC 3339] string     | [`DateTime`][s4]      |
-//! | [`Span`]            | [ISO 8601] duration   | [`Duration`][s5]      |
+//! | Rust type            | Format                | GraphQL scalar        |
+//! |----------------------|-----------------------|-----------------------|
+//! | [`civil::Date`]      | `yyyy-MM-dd`          | [`LocalDate`][s1]     |
+//! | [`civil::Time`]      | `HH:mm[:ss[.SSS]]`    | [`LocalTime`][s2]     |
+//! | [`civil::DateTime`]  | `yyyy-MM-ddTHH:mm:ss` | [`LocalDateTime`][s3] |
+//! | [`Timestamp`]        | [RFC 3339] string     | [`DateTime`][s4]      |
+//! | [`Zoned`][^1]        | [RFC 9557] string     | `ZonedDateTime`       |
```

Comment:
@sgoll I cannot come up with better naming. I think it's OK. Once such scalar will appear, we always can rename and adjust it.

Reply:
`ZonedDateTime` is indeed probably the right name for this. For example, [it's what Temporal uses](https://tc39.es/proposal-temporal/docs/zoneddatetime.html). It's also my second choice for Jiff, but I decided on a shorter name since "jiff" already provides the datetime context in the name. (And I prefer more concise names when it isn't too opaque.)